saleor/payment/gateways/adyen/plugin.py

```diff
@@ -478,10 +481,17 @@ def process_payment(
             currency=payment_information.currency,
             transaction_id=result.message.get("pspReference", ""),
             error=error_message,
+            # @deprecated
             raw_response=result.message,
             action_required_data=action,
             payment_method_info=payment_method_info,
             psp_reference=psp_reference,
+            legacy_adyen_plugin_payment_method=result.message.get("paymentMethod", "")
+            .strip()
+            .lower(),
```

Comment:
Suggestion (non-blocking): you could create a helper for cleaning the values `strip() and lower()` as it's used multiple times

Reply:
I added it to main, because it will be actively maintained